schemas/documents/www/homePageWeb.js

```diff
@@ -9,6 +9,12 @@ export default {
       title: 'Fold Section',
       validation: Rule => Rule.required(),
     },
+    {
+      name: 'carbonPlusSection',
```

Comment:
this is a bit weird to still name this "carbonPlusSection" while it's now more about the marketplace, but we do already have a "marketplaceSection" in here so not sure which naming would be best...

Reply:
I agree this is a bit weird. This section could be called `marketplaceIntroSection` and the other one `marketplaceActorsSection`. That being said would be best to do this refactoring in another issue since we estimated this one to 2.